web/tests/settings_user_groups_legacy.test.js

```diff
@@ -306,7 +306,11 @@ test_ui("populate_user_groups", ({mock_template, override, override_rewire}) =>
             assert.ok(get_by_email_called);
             assert.equal(typeof res, "object");
             assert.equal(res.user_id, bob.user_id);
-            assert.equal(res.display_value, bob.full_name + " (deactivated)");
+            assert.equal(
+                res.display_value,
+                bob.full_name +
+                    ` <i class="fa fa-ban deactivated-user-icon tippy-zulip-delayed-tooltip" data-tippy-content="User is deactivated"></i>`,
```

Comment:
For all of the tooltips, we need to present them with translation strings--not the hard-coded English "User is deactivated"